index.html

```diff
@@ -87,10 +87,11 @@
 
 <aside class="example" title="Measuring impact" id="example-measuring">
 
-Example: if a local government decides to add separated bicycle lanes and
-signalling then they should expect fewer bicycle accidents and more use of
-bicycles.  They can measure that by collecting automated non-identified traffic
-data and by monitoring published statistics about bicycle accidents.
+Example: If a local government decides to add separated bicycle lanes and
+improved signalling, they should expect increased bicycle use and a reduction in
+collisions involving bicycles. They can measure this by collecting automated,
+non-identified traffic data and by analysing transportation safety statistics,
+while being mindful of how such incidents are categorised in official records.
```

Comment:
```suggestion
while being mindful of how such incidents are categorised in official records, e.g., misclassified as bicycle accidents instead of automobile accidents or grouped under general traffic accidents.
```